src/olympia/constants/reviewers.py

```diff
@@ -210,3 +210,14 @@
     (AUTO_APPROVED, 'Was auto-approved'),
     (NOT_AUTO_APPROVED, 'Was *not* auto-approved'),
 )
+
+# Types of Add-ons for Reasons.
+REASON_ADDON_TYPE_ALL = 1
+REASON_ADDON_TYPE_EXTENSION = 2
+REASON_ADDON_TYPE_THEME = 3
```

Comment:
```suggestion
REASON_ADDON_TYPE_ALL = 0
REASON_ADDON_TYPE_EXTENSION = 1
REASON_ADDON_TYPE_THEME = 2
```
Or, if you changed the theme constant to `10` they'd actually be the same as base `ADDON_` constants. (0 is `ANY`; 1 is `EXTENSION`)

Reply:
Thanks. I think it makes sense to just reuse `ADDON_ANY`, etc., so I've done that.